itl/plugins-contrib.d/network-components.conf

```diff
@@ -833,3 +833,99 @@ object CheckCommand "printer_health" {
 	vars.printer_health_hostname = "$check_address$"
 	vars.printer_health_mode = "supplies-status"
 }
+
+object CheckCommand "fiberchannel-brocade" {
+	import "ipv4-or-ipv6"
+	command = [ PluginContribDir + "/check_snmp_brocade" ]
+	arguments = {
```

Comment:
Please add also here some empty lines as in the already existing CheckCommand definitions...

Reply:
Added, thanks!